olive/systems/system_config.py

```diff
@@ -147,7 +147,15 @@ def validate_config_type(cls, values):
         if system_alias_class:
             values["type"] = system_alias_class.system_type
             if system_alias_class.accelerators:
-                values["config"]["accelerators"] = [{"device": acc} for acc in system_alias_class.accelerators]
+                valid_accelerators = []
+                for device in system_alias_class.accelerators:
+                    valid_accelerators.extend(
+                        {"device": acc["device"], "execution_providers": acc.get("execution_providers")}
+                        for acc in values["config"]["accelerators"]
+                        if acc["device"].lower() == device.lower()
+                    )
+
+                values["config"]["accelerators"] = valid_accelerators or None
```

Comment:
Is it possible to remove this line?

Reply:
for AzureML alias, the accelerators including the ep must be provided.